src/Controller/SentencesController.php

```diff
@@ -1130,15 +1131,16 @@ public function edit_license()
         } else {
             $errors = $sentence->getError('license');
             $savedSentence = $this->Sentences->save($sentence);
+            $licenseName = Licenses::allLicenses()[$newLicense]['name'] ?? $newLicense;
             if ($savedSentence) {
                 $this->Flash->set(format(
-                    __('The license of the sentence has been changed to “{newLicense}”.'),
-                    compact('newLicense')
+                    __('The license of the sentence has been changed to “{licenseName}”.'),
+                    compact('licenseName')
                 ));
             } elseif (!empty($errors)) {
                 $message = format(
-                    __('Unable to change the license to “{newLicense}” because:'),
-                    compact('newLicense')
+                    __('Unable to change the license to “{licenseName}” because:'),
+                    compact('licenseName')
```

Comment:
Could you change this so that the translatable strings are not modified? Only the placeholder name is changed, so UI translators will have to add a new translation for no reason.

Reply:
Sorry, I've forgot about the translators.